tools/testsys/src/aws_resources.rs

```diff
@@ -191,6 +180,13 @@ pub(crate) async fn ec2_crd(
                 .testsys_agent_pull_secret
                 .clone(),
         )
+        .device_mappings(Some(
+            bottlerocket_input
+                .crd_input
+                .config
+                .block_device_mapping
+                .clone(),
+        ))
```

Comment:
This is safe if the config is not set? Does this just result in `Some()` of an empty Vec?